validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java

```diff
@@ -143,6 +146,10 @@ public Bytes32 getGraffiti() {
     return graffiti;
   }
 
+  public Path getGraffitiFile() {
```

Comment:
We're not currently using this setter.  This will need to be hooked up in [`ValidatorOptions`](https://github.com/ConsenSys/teku/blob/8a21fe768c1f30d2df0b3b8742baa0144b4082ab/teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java#L28-L35).